src/app/retrospective/sprint-list/sprint-list.component.ts

```diff
@@ -16,12 +16,29 @@ import { UtilsService } from '../../shared/utils/utils.service';
     styleUrls: ['./sprint-list.component.scss']
 })
 export class SprintListComponent implements OnInit {
-    @Input() retrospectiveID: any;
-    displayedSprintColumns = ['title', 'start_date', 'end_date', 'status', 'created_by', 'last_synced_at'];
-
+    lastScrollTop: number;
     dataSource: SprintListDataSource;
+
     sprintStates = SPRINT_STATES_LABEL;
     dateFormat = DATE_FORMAT;
+    displayedSprintColumns = ['title', 'start_date', 'end_date', 'status', 'created_by', 'last_synced_at'];
+
+    @Input() retrospectiveID: any;
+
+    @HostListener('window:scroll', [])
+    onWindowScroll() {
+        const currentHeight = window.innerHeight + window.scrollY;
+        const minScrollHeight = document.body.offsetHeight * LOAD_NEXT_PAGE_SCROLL_POSITION;
+
+        // If the user has scrolled to 70% of the page and is scrolling down, add more data
+        if (currentHeight > minScrollHeight && window.pageYOffset > this.lastScrollTop) {
```

Comment:
check if we have any good packages to support infinite scrolling. Also, if not found, we should develop this as a re-usable service/components

Reply:
@anmolghosh Added a TODO for the same and creating a ticket also